api/clients/codecs/default_blob_codec.go

```diff
@@ -22,7 +22,7 @@ func (v DefaultBlobCodec) EncodeBlob(rawData []byte) ([]byte, error) {
 	codecBlobHeader := make([]byte, 32)
 	// first byte is always 0 to ensure the codecBlobHeader is a valid bn254 element
 	// encode version byte
-	codecBlobHeader[1] = byte(DefaultBlobEncoding)
+	codecBlobHeader[1] = byte(PayloadEncodingVersion0)
```

Comment:
doesn't this function `EncodedBlob` need to be changed? it returns []byte, shouldnt it return a Blob in your type system? Or whatever else this is (prob just an encodedPayload?)

Reply:
`default_blob_codec` will not be used in v2, all encoding is contained in the new structs

Reply:
I see... shouldnt we separate v1 and v2 codecs then? Why are they all in the same package? Or am I missing something?

Reply:
Yeah, very good point, done in (3db92747)[https://github.com/Layr-Labs/eigenda/pull/1120/commits/3db927475247142dbcd7e715a04cc5d199ea1fde)

Unfortunately, I made the mistake of sharing code between v1 and v2 before I knew better. So, `polynomial_form.go` still lives in the `codecs` package. Everything else v2 related is now in the package `codecs/v2`, though

Reply:
Nice, this looks cleaner! One last nit... I really don't like when /v2 is the last suffix, because it makes the package name "v2". So if we have that across the repo and import a bunch of "v2"s, they all have to be renamed. The convention is to put /v2 before the actual package name.

haven't verified that it makes sense here, but would .../v2/codecs/ work?

Reply:
You're right, this is a much better configuration, obvious in hindsight [b8d43f59](https://github.com/Layr-Labs/eigenda/pull/1120/commits/b8d43f59ba3c666d9c8946770c87b11cb9c0c04e)